### ISSUE TITLE
Gitbox v1

### ISSUE THREAD

**Issue by jeff-h** [oleganza/gitbox2#1]
I suspect there's a loyal band of Gitbox v1 fans who prefer it over any other git GUI, despite how plentiful they have become on macOS. Gitbox remains utterly unique in its interpretation of git concepts into its UI/UX. It just makes sense, and once you have that viewpoint, all the other apps are horrible :) They all feel like a near-1:1 mapping from the git command line.

Given that you have indicated you're philosophically willing to open source Gitbox (as indicated by this repository), I am wondering if you would consider open-sourcing Gitbox v1.

I recognise its codebase is unlikely to reflect your ideals of 2022, but open sourcing it would, I'd hope, allow us to fix the small number of niggles that it's beginning to exhibit, as well as bundle an up-to-date git binary. All-in-all it's quite remarkable to me how little its broken over the many macOS releases since 2012.

**Comment by soniktrooth** [comment]
I second this! Every other git gui is an overcomplicated UI nightmare. Long live Gitbox!

**Comment by marcocanestrari** [comment]
Yes please! I'm a very happy and loyal Gitbox user since the very very beginning!
